core/adminserver/admin_server_test.go

```diff
@@ -37,6 +38,10 @@ func vrfKeyGen(ctx context.Context, spec *keyspb.Specification) (proto.Message,
 }
 
 func TestCreateRead(t *testing.T) {
+	// We can only run the integration tests if there is a MySQL instance available.
+	if provider := testdb.Default(); !provider.IsMySQL() {
+		t.Skipf("Skipping map integration test, SQL driver is %v", provider.Driver)
```

Comment:
I'm beginning to think this idiom is dangerous.  It's too easy to forget to set the relevant environment variable then be persuaded all is OK when the test silently passes.
What about t.Errorf() instead?

Reply:
I agree with you. @codingllama what do you think?   If we make it error, we should probably make it error everywhere.